### PR TITLE
[miio] suggest correct devicenames in readmehelper

### PR DESCRIPTION
**Pull request by marcelrv**
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json [hunk 1]
```diff
@@ -0,0 +1,2725 @@
...
+  "aden.aircondition.a1": "VINO Inverter Air Conditioner(Energy Label Level 3)",
+  "aden.aircondition.a2": "VINO Inverter Air Conditioner(Energy Label Level 1)",
+  "aden.aircondition.a4": "VINO Inverter Air Conditioner",
+  "afour.s_lamp.001": "ZIWOOO Intelligent germicidal lamp",
+  "aice.motor.kzmu3": "KMZ intelligent shutter door controller",
+  "aihome.light.m85": "light",
+  "aihome.light.m93": "FanLight",
+  "aiicn.curtain.r51": "智能窗帘控制器",
+  "aiicn.light.pwm2": "LED驱动器",
+  "aiicn.light.r01": "单路接收器",
```
Comment (cpmeister) [inline bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json:81]: I'm not sure that chinese names would really help english speakers know what they are looking at...

**Review by cpmeister — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json:81]: I'm sure not... these devices don't have an English description in the cloud, guess they are only for sale in CN.
This list is all devices that are known for Xiaomi, that does not mean they are included in OH..

Dunno an easy fix for this (I can delete all non-english ones, but that does not solve anything)

Comment (cpmeister) [reply bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json:81]: I'm not really sure there is a good fix either. But I think as long as you only show the device name as additional information for the device and not as the key identifier for the device, both in openHAB and in documentation, then I think we should be ok.

Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/resources/misc/device_names.json:81]: When I actually implement such a device I normally use the English names. Until very recent I didn't even know what is the name of a device according to Xiaomi. I used a mix of commercial names ( how they call the devices on the webshops) or generic names based on the model e.g aiicn.light.r01 would become aiicn light, and it still unclear Google translate as final solution

Long story short... When I actually add a device, I'll add it with an English name.
The inclusion here in the readme maker is just to speedup/simplify the creation of the actual Java devicelist file.

**Review by cpmeister — APPROVED** [review]

**Pull request merged**